src/main/java/md/forum/forum/services/LikeService.java

```diff
@@ -0,0 +1,59 @@
+package md.forum.forum.services;
+
+import md.forum.forum.models.Comment;
+import md.forum.forum.models.Like;
+import md.forum.forum.models.Post;
+import md.forum.forum.models.User;
+import md.forum.forum.repository.CommentRepository;
+import md.forum.forum.repository.LikeRepository;
+import md.forum.forum.repository.PostRepository;
+import md.forum.forum.repository.UserRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.Optional;
+
+@Service
+public class LikeService {
+    private final LikeRepository likeRepository;
+    private final UserRepository userRepository;
+    private final PostRepository postRepository;
+    private final CommentRepository commentRepository;
+
+    public LikeService(LikeRepository likeRepository, UserRepository userRepository, PostRepository postRepository, CommentRepository commentRepository) {
+        this.likeRepository = likeRepository;
+        this.userRepository = userRepository;
+        this.postRepository = postRepository;
+        this.commentRepository = commentRepository;
+    }
```

Comment:
Please replace with `@RequiredArgsConstructor`

Reply:
This is not a mistake, if @Liebe001 choose this method of initialization is his decision, this class doesn't have a lot of lines so I don't see a problem, @Liebe001 can you please comment on your decision ? Thx.